src/cli.rs

```diff
@@ -63,6 +63,10 @@ pub struct ExportGenesisStateCommand {
 	#[structopt(short, long)]
 	pub raw: bool,
 
+	/// Id of the parachain this state is for.
+	#[structopt(long)]
+	pub parachain_id: Option<u32>,
```

Comment:
I have removed this since we dont need it anymore for Genesis State

Reply:
Unfortunately polkadot-launch still expects it to be there - https://github.com/paritytech/polkadot-launch/blob/master/src/spawn.ts#L125

And generally I think it makes sense for our CLI to follow the conventions of other cumulus projects

Reply:
That was older projects. I think they are trying to be backward compatible. This would be dead code on our end since we upto latest